xbmc/ServiceManager.h

```diff
@@ -72,6 +72,7 @@ class CServiceManager
   CPlatform& GetPlatform();
 
   PLAYLIST::CPlayListPlayer& GetPlaylistPlayer();
+  int init_level = 0;
```

Comment:
Is there a reason for this being a public member?

Why is the member decl placed randomly in between member function's decl instead of having it where the other members are? 

And why has it such a "strange" name? Not 100% sure, but don't we usually use the same notation for members, would be <code>iInitLevel</code> in this case?